.circleci/config.yml

```
@@ -102,6 +111,9 @@ workflows:
       - unit_test:
           requires:
             - install
+      - chromatic:
+          requires:
+            - install
```

We should do a preview only if the tests pass
```suggestion
            - unit_test
```

---

You should probably remove the `install` requirement, it will be met by virtue of the unit tests requiring them. I think this is breaking the CI too, as it's not triggering the workflow?